pkgs/by-name/pg/pgraphs/package.nix

```diff
@@ -6,16 +6,16 @@
 
 buildNpmPackage rec {
   pname = "pgraphs";
-  version = "0.6.13";
+  version = "0.6.17";
 
   src = fetchFromGitHub {
     owner = "pg-format";
     repo = "pgraphs";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-NLQMBEN/wO/xOMy+gX3sQZRqU8gYesXS7hwRGWyjvX0=";
+    rev = "v${version}";
```

Comment:
No, we want to keep refs/tags as that makes the reference unique across tags and branches.

Reply:
Applied suggestion